trie/keyBuilder/keyBuilder.go

```diff
@@ -0,0 +1,56 @@
+package keyBuilder
+
+import (
+	"github.com/ElrondNetwork/elrond-go/common"
+)
+
+// NibbleSize marks the size of a byte nibble
+const NibbleSize = 4
+
+type keyBuilder struct {
```

Comment:
we need to refactor this as it will allocate a lot of memory hindering the performance
Not a great solution in mind for this, atm

Reply:
Will think about an improved solution.